### PR TITLE
get abolute url tests

### PR DESCRIPTION
**Pull request by Akiyamka**
This branch add tests for absolute url generator, and fix behavoiur in edge cases
Don't merge it before release branch 2.2.6

### CODE REVIEW

File: src/core/localization/gettext/template/common.pot [hunk 1]
```diff
@@ -5,8 +5,8 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2022-12-21T10:53:46.140Z\n"
-"PO-Revision-Date: 2022-12-21T10:53:46.140Z\n"
+"POT-Creation-Date: 2022-12-21T13:31:44.734Z\n"
```
Comment (propakov) [inline src/core/localization/gettext/template/common.pot]: it shouldn't be in PR

File: src/core/router/__snapshots__/getAbsoluteRoute.test.ts.snap [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// Vitest Snapshot v1
```
Comment (propakov) [inline src/core/router/__snapshots__/getAbsoluteRoute.test.ts.snap:1]: Cool! I didn't know about such approach! 😎

**Review by propakov — APPROVED** [review]

File: src/core/router/getAbsoluteRoute.ts [hunk 3]
```diff
@@ -1,13 +1,7 @@
...
 
 const trimSlash = (str: string) => {
-  if (str.startsWith('/')) {
-    str = str.slice(1);
-  }
-  if (str.endsWith('/')) {
-    str = str.slice(0, -1);
-  }
-  return str;
+  return str.replace(/^\/|\/$/g, '');
```
Comment (IlyaIzr) [inline src/core/router/getAbsoluteRoute.ts:5]: here you can add comment of what this regex does
